src/bindings/python/flux/util.py

```diff
@@ -30,8 +30,10 @@
 
 import yaml
 
+# tomllib added to standard library in Python 3.10
```

Comment:
```suggestion
# tomllib added to standard library in Python 3.11
```

Reply:
yeah, i got conflicting information about 3.10 vs 3.11.  Seems we got it wrong in this prior commit too :-(  6298a362324f03700362e44990ab212f12a5374c.  But googling it does seem it is 3.11.

Reply:
Yeah, oops!